.gitignore

```diff
@@ -278,3 +278,6 @@ Session.vim
 # Private test configuration and binaries.
 config.ps1
 **/IISApplications
+
+# VS debug support files
+launchSettings.json
```

Comment:
Do we really want to git ignore ALL `launchSettings.json` files in the whole repo? Will it ever be possible that some other kind of project will want to check in a `launchSettings.json` file?

Reply:
I think were fine for now and if we ever want to check one in we can change the rules.